resources/js/features/games/components/AchievementSetEmptyState/AchievementSetEmptyState.tsx

```diff
@@ -44,7 +44,7 @@ export const AchievementSetEmptyState: FC = () => {
               }}
               components={{
                 // eslint-disable-next-line jsx-a11y/anchor-has-content -- this is passed in by the consumer
-                1: auth?.user ? <a href={`/setRequestors.php?g=${backingGame.id}`} /> : <span />,
+                1: auth?.user ? <a href={`/game/${backingGame.id}/requests`} /> : <span />,
```

Comment:
Performance: now that the page is using Inertia, we should use an `<InertiaLink />` component here to ensure a client-side navigation.